modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java

```diff
@@ -2202,6 +2386,18 @@ private MockLiferayPortletRenderRequest _getMockLiferayPortletRenderRequest(
 			"getSearchContainer", new Class<?>[0]);
 	}
 
+	private ServiceContext _getServiceContext(
```

Comment:
Usually groupId always go first https://github.com/liferay/liferay-portal/blob/11e6081f96abb6bf299369519434c1eafa0658e3/portal-kernel/src/com/liferay/asset/kernel/service/AssetEntryLocalService.java#L599

Usually we would use

userId, groupId, assetCategoryIds

Reply:
Yes, that's true. But there are still inconsistencies using  `userId, groupId, assetCategoryIds`, specially creating a service context, see [getServiceContext](https://github.com/liferay/liferay-portal/blob/master/portal-test/src/com/liferay/portal/kernel/test/util/ServiceContextTestUtil.java#L51). Some time ago I decided to go always alphabetically. So, if you don't mind i'm going to forward it as it is. Is it ok?

Reply:
We should try to follow Liferay conventions, an groupId, usually go first, if you think we should always order alphabetically, open the suggestion in the sf channel, so we can ask Hugo to include this rule.

Reply:
Perfect, let's follow the conventions, thanks @cgoncas

Reply:
actaully, after the change it looks more natural for me, let's see what brian thinks